### PR TITLE
feat(ls): Ignore findings locally

### PR DESCRIPTION
**Pull request by ajbt200128**
This PR allows users to ignore findings in their code (based off match based ID) without using # nosem or triage ignoring it in the app.

## Test plan
Wrote some tests in Test_LS_e2e

Also tested manually in VSCode

### CODE REVIEW

File: changelog.d/pdx-154.added [hunk 1]
```diff
@@ -0,0 +1 @@
+Users can now ignore findings locally in Semgrep IDE Extensions
```
Comment (aryx) [inline changelog.d/pdx-154.added]: maybe say where is the information saved? (In the "session"?)
If I reboot is the information saved?

File: src/osemgrep/language_server/requests/Execute_command.mli [hunk 2]
```diff
@@ -0,0 +1,5 @@
+val handle_execute_request :
```
Comment (aryx) [inline src/osemgrep/language_server/requests/Execute_command.mli:1]: please add comment explaining what this function is doing.
Its name is very vague.

File: src/osemgrep/language_server/server/Session.ml [hunk 3]
```diff
@@ -216,30 +218,75 @@ let fetch_skipped_fingerprints () =
...
+  List.sort_uniq String.compare skipped_fingerprints
+
 let runner_conf session =
   User_settings.core_runner_conf_of_t session.user_settings
 
 let previous_scan_of_file session file =
   Hashtbl.find_opt session.cached_scans file
 
+let save_local_skipped_fingerprints session =
+  let save_dir = !Env.v.user_dot_semgrep_dir / "cache" / "fingerprints" in
```
Comment (aryx) [inline src/osemgrep/language_server/server/Session.ml]: maybe fingerprinted_findings?
or the one that are ignored? fingerprinted_ignored_findings?

**Review by aryx — APPROVED** [review]

File: src/osemgrep/language_server/server/Session.ml (hunk 3 above)
Comment (ajbt200128) [reply src/osemgrep/language_server/server/Session.ml]: Sure

File: changelog.d/pdx-154.added (hunk 1 above)
Comment (ajbt200128) [reply changelog.d/pdx-154.added]: Will update

**Pull request merged**